joomla/joomla-cms
# Add Sort by Modified Date in the Module mod_articles_news

Pull Request for Issue # .  This Pull Request is not based on any issue.  It is an improvement to the existing function of the module.  At present the module has the facility to sort the articles by the publish up and created dates.  With this change, the ability to sort the articles by the modified date is also possible.  Especially, this change would help in listing the recently modified articles.
#### Summary of Changes

(a) File mod_articles_news.xml -> Added another sort option value a.modified (Article Modified Date) to the existing values a.publish_up (Article Publish Start/Up Date) and a.created (Article Created Date). 
(b) File mod_articles_news.ini -> Added the new language string to show the text string "Modified Date".
#### Testing Instructions

Modify a couple of articles that has the earliest publish up date.  Change the sort order in the module to "Modified Date" in the option "Order Results".  You should see list of modified articles.
